frappe/sessions.py

```diff
@@ -171,6 +171,7 @@ def get():
 		frappe.get_attr(hook)(bootinfo=bootinfo)
 
 	bootinfo["lang"] = frappe.translate.get_user_lang()
+	bootinfo["translated_search_doctypes"] = frappe.get_hooks("translated_search_doctypes")
```

Comment:
Not yet, but it will be useful for translating link fields.

Reply:
Can you explain with an example? 😅
Perhaps, if this will be useful after another PR, it can be added there directly?

Reply:
Added the related PR to my previous comment. The link control needs to know if the values will be translated because it behaves differently in this case. Until now the values are hardcoded over there.